polyfactory/value_generators/constrained_collections.py

```diff
@@ -29,11 +31,14 @@ def handle_constrained_collection(
     :param max_items: Maximal number of items.
     :param min_items: Minimal number of items.
     :param unique_items: Whether the items should be unique.
+    :param field_build_parameters: Any build parameters passed to the factory as kwarg values.
+    :param build_context: BuildContext data for current build.
 
     :returns: A collection value.
     """
     min_items = abs(min_items if min_items is not None else (max_items or 0))
     max_items = abs(max_items if max_items is not None else min_items + 1)
+    max_unique_value_attempts = 5
```

Comment:
@guacs I am also not sure about this. Please pay attention 🙏 
 - Should I keep unique_attemps default value equal to 5. `Faker` for example has similar code for generating unique items , but have more default attempts `1000` https://github.com/joke2k/faker/blob/db5a5ca9ef1dc9450a2a309945a7faa75007febb/faker/proxy.py#L18 
 - Should I try to read this value from build_context?

Reply:
I'd favour doing this in a separate PR if possible. This is changing behaviour rather than a fix.

I think the suggested behaviour makes sense with configuration via a variable on `Factory` itself as available here

Reply:
I agree, I reverted the code connected to this.